pkg/interface/link/src/js/components/root.js

```diff
@@ -93,44 +122,47 @@ export class Root extends Component {
                   comments={channelComments}
                   seen={channelSeen}
                   page={page}
-                  groupPath={groupPath}
+                  resourcePath={resourcePath}
+                  resource={resource}
                   popout={popout}
                   sidebarShown={state.sidebarShown}
                   />
                 </Skeleton>
               )
             }}
           />
-          <Route exact path="/~link/(popout)?/:ship/:channel/:page/:index/:encodedUrl/(comments)?/:commentpage?"
+          <Route exact path="/~link/(popout)?/item/:page/:index/:commentpage/:encodedUrl/:resource([^#]+)"
```

Comment:
Er, I'm actually now confused as to what resource correlates to; if it's a collection path that the item belongs to, this is definitely out of order.

Reply:
Yes, it's a collection path. But it's a collection _path_. You can't put that in the middle of another path and parse it out sanely. So if you ever have multi-part paths there (ie `/webdev/web1`, you kind of need it to be at the end.

cc @loganallenc (re "weird path format")

Reply:
Seems strange to me too btw. I don’t know the data structures of links well enough to understand what each item corresponds to, but this is... certainly unconventional.

Reply:
> But it's a collection path. You can't put that in the middle of another path and parse it out sanely.

This is why Chat, Notebooks, etc don't say `:path` params but `:ship/:name` and why I am starting to think we should disallow slashes in names — I think the most sane, consistent pattern we can do across all apps is dictate static identifiers that follow `host/group/channel` nesting. I'm a bit unsettled that we are deliberating this now.

Reply:
I'm all for reducing the degrees of freedom wherever possible here; especially where there is precedent with chat and notebooks. FWIW the multi-channel per group functionality will cover what (ostensibly) the use case of multi-segment path collection IDs.

Let's make sure we don't leave standup today without putting this to bed.

Reply:
As per standup: it isn't that important but I'll go do the consistent thing and make frontend-side assumptions about what resource paths look like.